src/Gelf/Transport/TcpTransport.php

```diff
@@ -27,28 +26,52 @@
  */
 class TcpTransport extends AbstractTransport
 {
-    const DEFAULT_HOST = "127.0.0.1";
-    const DEFAULT_PORT = 12201;
+    /**
+     * @var string
+     */
+    protected $host = "127.0.0.1";
+
+    /**
+     * @var int
+     */
+    protected $port = 12201;
 
     /**
      * @var StreamSocketClient
      */
     protected $socketClient;
 
+    /**
+     * @var SslOptions|null
+     */
+    protected $sslOptions = null;
+
     /**
      * Class constructor
      *
-     * @param string $host      when NULL or empty DEFAULT_HOST is used
-     * @param int    $port      when NULL or empty DEFAULT_PORT is used
+     * @param string|null     $host       when NULL or empty default-host is used
+     * @param int|null        $port       when NULL or empty default-port is used
+     * @param SslOptions|null $sslOptions when null not SSL is used
      */
-    public function __construct($host = self::DEFAULT_HOST, $port = self::DEFAULT_PORT)
+    public function __construct($host = null, $port = null, SslOptions $sslOptions = null)
```

Comment:
What's your reasoning for removing the default constants and use nullable types? In my opinion it's better to have strict (as in not-nullable) types and the aforementioned constants